#Fix for rwiki wrong bodies
#Cherry Pick
=svnm= https://source.sakaiproject.org/svn/rwiki/trunk@49345 rwiki

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-14076.patch 
@@ -0,0 +1 @@
+ 
